pkg/netns/netns_windows.go

```diff
@@ -0,0 +1,50 @@
+// +build windows
+
+/*
+Copyright The containerd Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netns
+
+// TODO(windows): Implement netns for windows.
+// NetNS holds network namespace.
+type NetNS struct {
```

Comment:
I keep wondering if we should just make this an interface rather than a hard type. What do you think?

Reply:
It doesn't matter right now. :)

We can add it whenever needed.

Reply:
Sounds good.